client/src/styles/components/bootstrap.form.css

```diff
@@ -20,3 +20,12 @@ textarea.form-control {
 .input-group-prepend {
   z-index: 1; /* so child button border is above sibling input border */
 }
+
+.form-control {
+  /* To increase height jsut enough to match height of regular _common Button */
+  height: calc(1.5em + 0.75rem + 3px); /* 3px ← 2px from Bootstrap _forms.css */
```

Comment:
@wesleyboar I was seeing that this change is overwriting the DropdownSelector `height`, leading to [this issue](https://github.com/TACC/Core-Portal/pull/662#discussion_r947352633) Nathan found. What do you think?

Reply:
Ah. I see. I think I did that. 😅

I propose a new PR (off of main after this is merged) to deal with it. I will create one with my local test code.

Nathan says this got fixed. What was your fix? (If it's temporary that is fine.)

<details>

I would update `<DropdownSelector>` to accept `size` prop (like `<Button>`) and translate the sizes to [Reactstrap `<Input>`'s `bsSize` prop](https://6-4-0--reactstrap.netlify.app/components/form/#Input-Sizing).

I tested it on Data Files search. (I had to add a test button next to it, and tell flexbox not to stretch the button nor input.) I don't how wide the effects are, and don't want to hold up this PR, so I'd rather such a solution be a new PR.

</details>

Reply:
Does the new ticket, [FP-1806](https://jira.tacc.utexas.edu/browse/FP-1806): Project Role Selector and Button Too Big, accurately describe the problem we have if we merge as is?

Reply:
Yes, that new ticket looks good!